pages/blog/documenting-sqs-event-driven-apis-with-asyncapi.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Documenting SQS event-driven APIs with AsyncAPI
+date: 2022-08-18T12:40:20-03:00
+type: Engineering
+tags: ['Getting Started','SQS']
+cover: /img/posts/documenting-sqs-event-driven-apis-with-asyncapi/cover.webp
+authors:
+  - name: Mauro Bosetti
+    photo: /img/avatars/mbosetti.webp
+    link: https://www.linkedin.com/in/mauro-bosetti/?locale=en_US
+    byline: Argentinian software explorer.
+excerpt: Documenting event-driven architectures is possible, and this post will show you how, using SQS as a (real world) example.
+---
```

Comment:
```suggestion
featured: true
---
```

Reply:
feel free to add this metadata that will make your article featured at the top

Reply:
Thanks for the suggestion!